charts/victoria-metrics-alert/values.yaml

```diff
@@ -108,6 +108,7 @@ server:
   # Multiple notifiers can be enabled by using `notifiers` section
   notifier:
     alertmanager:
+      config: ""
```

Comment:
Should the `config` be outside of `alertmanager`? Since it didn't belong to any alertmanager config.

Reply:
@Haleygo All right, I did it. Can you verify again? Thanks.